### DISCUSSION TITLE
unable to login grafana

### DISCUSSION THREAD

**Discussion by devilathag** [learndevopsonline/learndevopsonline#1511] [Q&A]
Dear Team,

  I am unable to login grafana due to invalid username or password, i thought i didn't put any password for this
 
<img width="907" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/144150880/888ce972-6718-4934-9188-8bf40215440f">

**Comment by sreedharm07** [comment]
Hi @devilathag 
if you are logging in for a first time to grafana user name and password will be admin and admin 
if its is not first time, when you logged in for first time it will ask to create new password, you should enter that new password.
if you forgot that delete machine and re create

**Comment by devilathag** [comment]
Thanks, I'll check it out.
